screen-pop-app/screen-pop/src/grantLogin.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState, useEffect, useRef } from "react";
+import axios from "axios";
+import "./App.css";
+import { Channel } from "./notification";
+import { ConfigData,  ConversationManager } from "./ConversationManager";
+
+const CLIENT_ID = "81a7fe1e-497b-4b99-aef0-eb67192750e9";
+const ENVIRONMENT = "mypurecloud.com"
+
+function getParameterByName(name: string) {
+  // eslint-disable-next-line
+  name = name.replace(/[\[]/, "\\[").replace(/[\]]/, "\\]");
+  var regex = new RegExp("[\\#&]" + name + "=([^&#]*)"),
+    results = regex.exec(window.location.hash);
+  return results === null
+    ? ""
+    : decodeURIComponent(results[1].replace(/\+/g, " "));
+}
+
+export const App = () => {
```

Comment:
Since this is in a file named `grantLogin.tsx` this should be named `GrantLogin`